### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by datalorax** [jkosie/lab1#1]
Thank all!

So you have two `.Rproj` files in the root directory here. You definitely will only ever want to have one `.Rproj` for each project. Otherwise you can run into working directory issues. It's not so bad if they're both at the same directory level, but generally it's best avoided anyway.

In terms of your code:

Not a big deal, and sort of just a style preference, but I would recommend adding `set class = "tbl_df"` as an additional argument to `import` so it imports as a tibble, rather than a standard data frame. I just think they're a lot nicer to work with.

For interactive work it's generally not that big of a deal, but it's best to avoid using `T` and `F` in place of `TRUE` and `FALSE`. See here for an example of why: https://www.r-bloggers.com/r-tip-avoid-using-t-and-f-as-synonyms-for-true-and-false/

As is, your area is actually overlaid on top of your line/smooth. I would actually put this as the base layer and put the line/smooth on top of that. You're also missing a few other subtleties like a  caption but overall it looks good.

The biggest problem I'm seeing at this point is no Commit 2/3, but I'll come back and look again later (still isn't due yet, after all).

**Comment by jkosie** [comment]
Thanks!

I just made some edits:
* deleted the extra r project
* added `set class = "tbl_df"`
* changed T/F to TRUE/FALSE
* put area as base layer and line/smooth on top; added caption

However, when I look at the commits I do see 2 and 3. I'm not sure what needs to be fixed there.

**Comment by datalorax** [comment]
Yeah I’m not sure but they are not showing up in the Rmd (or, obviously, the HTML). I’m wondering if has to do with the files not being merged correctly (or at all).

Thanks,
--
Daniel Anderson
Research Assistant Professor
Behavioral Research and Teaching
University of Oregon

**Comment by jkosie** [comment]
Maybe it had something to do with the extra R project? I didn't look before deleting that project, but (at least now) I see the commits in the Rmd, and they're there when I knit to html.

**Comment by datalorax** [comment]
Perfect! I’ll look at those commits now.

Thanks,
--
Daniel Anderson
Research Assistant Professor
Behavioral Research and Teaching
University of Oregon

**Comment by datalorax** [comment]
Okay, so here's the rest of your feedback:

I see you're loading the utf8 library... is there a reason that's needed? Generally I recommend against loading packages that aren't needed for the specific script because it can lead to conflicts among the functions.

I'd recommend trying to pipe things, whenever possible. So, for example,

```r
countscreen25 <- tweets %>% 
  count(screen_name) %>% 
  arrange(desc(n)) %>% 
  slice(1:25)
```

Everything else looks great! Well done!

**Comment by kgrove10** [comment]
Thank you for your advice and feedback!  I loaded the utf8 library because it came up as an error ("loadNamespace(name) : there is no package called ‘utf8’") when I was trying to use the Tidyverse (not sure why, but that is what stack exchange recommended to fix the error!).

**Comment by datalorax** [comment]
Ah… that would be because you didn’t have the package installed, and it’s apparently a dependency of one of the tidyverse packages. So it was good to install it, but you don’t actually need to load it.

**Comment by kgrove10** [comment]
Oh, that's good to know.  Thank you!
